examples/plotting/ImageAnimatorWCS_Example.py

```diff
@@ -0,0 +1,37 @@
+"""
+================
+ImageAnimatorWCS
+================
+
+This example shows a simple visualization for an ImageAnimatorWCS object
+"""
+import astropy
+
+import sunpy
+import sunpy.map as smap
+from sunpy.data.sample import AIA_171_IMAGE, AIA_193_IMAGE
+from sunpy.visualization.animator import ImageAnimatorWCS
+
+import matplotlib.pyplot as plt
+import numpy as np
+
+###############################################################################
+# Creating the Mapcube of the given images
+sequence = smap.Map(AIA_171_IMAGE, AIA_193_IMAGE, cube=True)
+sequence = sequence.as_array()
+
+###############################################################################
+# The last two axes are the image
+# The shape is (10, 4096, 4096)
+sequence = np.rollaxis(sequence, -1)
+
+wcs_input_dict = {
+    'CTYPE1': 'Time    ', 'CUNIT1': 's', 'CDELT1': 12,
+    'CTYPE2': 'HPLN-TAN', 'CUNIT2': 'arcsec', 'CDELT2': 0.6,
+    'CTYPE3': 'HPLT-TAN', 'CUNIT3': 'arcsec', 'CDELT3': 0.6}
```

Comment:
Some of these numbers look like magic. Better to use the metadata from the maps.
```python
from itertools import product
import astropy.units as u

wcs_input_dict = {f'{key}{n+1}': sequence.all_meta()[0].get(f'{key}{n}') for n, key in product([1, 2], ['CTYPE', 'CUNIT', 'CDELT'])}
t0, t1 = map(parse_time, [k['date-obs'] for k in sequence.all_meta()])
time_diff = (t1 - t0).to(u.s)
wcs_input_dict.update({'CTYPE1': 'Time', 'CUNIT1': time_diff.unit.name, 'CDELT1': time_diff.value})
```

Which returns:
```
{'CTYPE2': 'HPLN-TAN', 'CUNIT2': 'arcsec', 'CDELT2': 2.402792,
 'CTYPE3': 'HPLT-TAN', 'CUNIT3': 'arcsec', 'CDELT3': 2.402792,
 'CTYPE1': 'Time',     'CUNIT1': 's',      'CDELT1': 5.0700000000020395}
```

which is different to the numbers that you set up, and it's using the real numbers from the data.

Reply:
wow ! this is a good way